### PR TITLE
Move "Import NFTs" to Modal

### PR DESCRIPTION
**Pull request by NidhiKJha**
To make the Import process of NFTs more visually appealing from the home screen, we're moving this functionality into a modal.

Fixes [#704](https://github.com/MetaMask/MetaMask-planning/issues/704)

## Screencaps

https://github.com/MetaMask/metamask-extension/assets/39872794/00fc2492-87cf-4662-93e0-52d8927a918f

## Manual Testing Steps

- Go to NFT Tab
- Click Import NFT
- Import NFT Modal opens
- Import NFT

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/app/nfts-tab/nfts-tab.js [hunk 1]
```diff
@@ -60,7 +60,11 @@ export default function NftsTab({ onAddNFT }) {
         />
       ) : (
         <>
-          {isMainnet && !useNftDetection ? <NftsDetectionNotice /> : null}{' '}
+          {isMainnet && !useNftDetection ? (
+            <Box padding={4}>
+              <NftsDetectionNotice />
+            </Box>
+          ) : null}{' '}
```
Comment (darkwing) [inline ui/components/app/nfts-tab/nfts-tab.js:67]: Is the `{' '}` really needed?

**Review by darkwing — APPROVED** [review]
We should autoFocus the first form field -- otherwise great!

**Review by darkwing — CHANGES_REQUESTED** [review]
Oops, this should be Request changes.

File: ui/components/app/nfts-tab/nfts-tab.js (hunk 1 above)
Comment (NidhiKJha) [reply ui/components/app/nfts-tab/nfts-tab.js]: Updated here https://github.com/MetaMask/metamask-extension/pull/19806/commits/35fd2defb23542c07b9d56051004c569ee5200b5

**Comment by NidhiKJha** [comment]
> We should autoFocus the first form field -- otherwise great!

Updated here https://github.com/MetaMask/metamask-extension/pull/19806/commits/35fd2defb23542c07b9d56051004c569ee5200b5

**Review by darkwing — APPROVED** [review]

**Review by vthomas13 — APPROVED** [review]

**Pull request merged**